eppo_client/client.py

```diff
@@ -183,114 +185,63 @@ def get_assignment_variation(
         """
         validate_not_blank("subject_key", subject_key)
         validate_not_blank("flag_key", flag_key)
-        experiment_config = self.__config_requestor.get_configuration(flag_key)
-        override = self._get_subject_variation_override(experiment_config, subject_key)
-        if override:
-            if expected_variation_type is not None:
-                variation_is_expected_type = VariationType.is_expected_type(
-                    override, expected_variation_type
-                )
-                if not variation_is_expected_type:
-                    return None
-            return override
+        if subject_attributes is None:
+            subject_attributes = {}
 
-        if experiment_config is None or not experiment_config.enabled:
-            logger.info(
-                "[Eppo SDK] No assigned variation. No active experiment or flag for key: "
-                + flag_key
-            )
+        flag = self.__config_requestor.get_configuration(flag_key)
+
+        if flag is None:
+            logger.info("[Eppo SDK] No assigned variation. Flag not found: " + flag_key)
```

Comment:
This might deserve a `warn`? (Broader point - if we were to differentiate between inactive and non-existing flags, this could be a good place to do so. Null because a flag is inactive is expected; null because a flag doesn't exist at all is a warning sign.)